src/pal/src/file/path.cpp

```diff
@@ -1215,12 +1191,12 @@ SearchPathA(
     
             PathLength = pPathEnd-pPathStart;
     
-            if (PathLength+FileNameLength+1 >= MAX_LONGPATH) 
+            if (PathLength+FileNameLength+1 >= MAX_LONGPATH) //Check if we have to do anything special 
             {
                 /* The path+'/'+file length is too long.  Skip it. */
                 WARN("path component %.*s is too long, skipping it\n", 
                      (int)PathLength, pPathStart);
-                continue;
+                //continue;
```

Comment:
It seems we don't need to skip long segments anymore, since we can handle any path length now. So you can delete this part.